java-shared-dependencies/first-party-dependencies/pom.xml

```diff
@@ -92,6 +92,13 @@
         <version>2.20.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-core:current} -->
         <classifier>tests</classifier>
       </dependency>
+      <dependency>
```

Comment:
I don't think we need it now that we use `${google.auth.version}` in showcase pom.

Reply:
removed